scss/_variables.scss

```diff
@@ -488,9 +488,11 @@ $border-color:                $gray-300 !default;
 // scss-docs-end border-variables
 
 // scss-docs-start border-radius-variables
-$border-radius:               .25rem !default;
-$border-radius-sm:            .2rem !default;
-$border-radius-lg:            .3rem !default;
+$border-radius:               .4rem !default;
```

Comment:
I am not sure, but this is the base variables file. Right? 
If we change the variables here, wouldn't be a breaking change, for dist files and for all that haven't override the radius variables?

Reply:
I'm changing the default values, but I don't see this as a breaking change. Going for a visual refresh on a few components alongside our docs.